corehq/apps/data_dictionary/management/commands/populate_case_prop_groups.py

```diff
@@ -40,10 +40,14 @@ def remove_out_of_sync_prop_and_groups():
     properties_out_of_sync = (CaseProperty.objects
                             .filter(group_obj__isnull=False)
                             .filter(~Q(group_obj__name=F('group'))))
-    print("Remove out of sync groups for {} properties".format(len(properties_out_of_sync)))
+    changed_properties = [prop.name for prop in properties_out_of_sync]
+    print("Reset out of sync groups for {} properties".format(len(properties_out_of_sync)))
+    print(changed_properties)
```

Comment:
It'd be helpful to also report domain and case type here if the idea is that these might need following up on

Reply:
I have added them to the log for these also. [0bc6705](https://github.com/dimagi/commcare-hq/pull/32799/commits/0bc6705fba7078b61eedce72c7fdff76cf8d8109)